src/i18n/locales/en-US.json

```diff
@@ -257,6 +257,10 @@
       "selectVideoLabel": "Video Flavors"
     },
 
+    "subtitleTimeline": {
+      "timeline-tooltip": "Mini Timeline"
```

Comment:
- I don't think that `Mini Timeline` is a very good description of this interface element. The tooltip should explain what an element does or what it is for. Maybe use something like “Overview Timeline”?
- Given that the subtitle editor has two timelines, the identifier `subtitleTimeline.timeline-tooltip` doesn't explain which timeline this translation string is for.  Plus, it's mixing camel case and kebab case. Maybe use `subtitleTimeline.overviewTimelineTooltip`?